.jabbarc

```diff
@@ -0,0 +1 @@
+adopt@1.8.0-232
```

Comment:
With a `.jabbarc`, you can `jabba use` and not have to remember a specific version, and also `jabba use` will make it so that we have a stable java version across development environments without needing a container. It's like a `.nvmrc`